flow/src/main/java/com/sequenceiq/flow/controller/FlowController.java

```diff
@@ -19,6 +21,8 @@
 @InternalOnly
 public class FlowController implements FlowEndpoint {
 
+    private static final int INPUT_LIST_UPPER_LIMIT = 500;
```

Comment:
We should have this configurable so that it can be increased/decreased in future without any code changes

Reply:
I had a discussion with Mihaly. He said there is no external configuration file as of now and this can be left as is.